index.html

```diff
@@ -0,0 +1,394 @@
+                <br>
+                <input type="checkbox" id="sevenDay" name="sevenDay" onclick="uncheck(2)"/>
+                <label for="today">7 ngày</label>
+                <br>
+                <input type="checkbox" id="month" name="month" onclick="uncheck(3)"/>
+                <label for="today">1 tháng</label>
+              </div>
+              <div>
+                <input type="checkbox" id="learn" name="learn" />
+                <label for="learn">Learn</label>
+                <br>
+                <input type="checkbox" id="work" name="work" />
+                <label for="work">Work</label>
+                <br>
+                <input type="checkbox" id="goout" name="goout" />
+                <label for="goout">Goout</label>
+                <br>
+                <input type="checkbox" id="dochores" name="dochores" />
+                <label for="dochores">DoChores</label>
+              </div>
+            </div>
+            <input type="submit" value="Tạo" onclick="filter()">
+          </fieldset>
+        </div>
+        <br>
+        <br>
+        <h1>Danh sách công viêc</h1>
+        <table id="tb">
+          </table>
+    </div>
+    <br>
+    <div>
+        <label for="name">Nhập tên người làm</label>
+        <input type="text" id="name">
+        <br>
+        <br>
+        <label for="task">Nhập tên công việc</label>
+        <input type="text" id="task">
+        <br>
+        <br>
+        <label for="date">Nhập ngày làm việc</label>
+        <input type="date" id="date">
+        <br>
+        <br>
+        <input type="submit" value="Tạo" onclick="create()">
+        <input type="submit" value="Cập nhật" onclick="edit()">
+        <input type="text" hidden id="editID">
+    </div>
+</body>
+<script>
+
+function init() {
+    if (!localStorage.getItem("toDoList")) {
+        let arr = [{id:1,Name:'Giau',Task:'WWork',Date:'2023-03-04'},{id:2,Name:'Anh',Task:'Work',Date:'2023-03-05'}];
+        localStorage.setItem("toDoList",JSON.stringify(arr));
+    }
+}
+init();
+
+
+
+ function get() {
+  
+  let arr = JSON.parse(localStorage.getItem("toDoList"));
+  render(arr);
+}
+get();
+
+function create() {
+  let arr1 = JSON.parse(localStorage.getItem("toDoList"));
+  let name = (document.getElementById("name").value).toString();
+  let task = (document.getElementById("task").value).toString();
+  let date = (document.getElementById("date").value).toString();
+  let obj = { id:arr1.length+1, Name:name, Task:task, Date:date};
+  arr1.push(obj);
+  console.log(arr1);
+  localStorage.setItem("toDoList",JSON.stringify(arr1));
+}
+function del(id) {
+  let arr1 = JSON.parse(localStorage.getItem("toDoList"));
+  for (let i =0 ; i< arr1.length;i++)
+  {
+    if(arr1[i].id == id)
+    {
+        arr1.splice(i,1);
+        document.getElementById("tb").deleteRow(i+1);
+        i--;
+    }
+  }
+  console.log(arr1);
+  localStorage.setItem("toDoList",JSON.stringify(arr1));
```

Comment:
create a new function to do this action
function updateLocalStorage(dataName, value) {
    if(dataName) {
        localStorage.setItem(dataName,JSON.stringify(value));
    }
}